src/libraries/Microsoft.Extensions.Logging.Abstractions/ref/Microsoft.Extensions.Logging.Abstractions.cs

```diff
@@ -23,14 +23,14 @@ namespace Microsoft.Extensions.Logging
     }
     public partial interface IExternalScopeProvider
     {
-        void ForEachScope<TState>(System.Action<object, TState> callback, TState state);
-        System.IDisposable Push(object state);
+        void ForEachScope<TState>(System.Action<object?, TState> callback, TState state);
```

Comment:
should `TState` be nullable?

Reply:
Same as https://github.com/dotnet/runtime/pull/43892#discussion_r514370522